front/app/containers/MainHeader/Components/NavbarContent/DesktopNavbarContent/index.tsx

```diff
@@ -181,13 +181,14 @@ const DesktopNavbarContent = () => {
   const { data: authUser } = useAuthUser();
   const locale = useLocale();
   const theme = useTheme();
-  const windowSize = useWindowSize();
   const { formatMessage } = useIntl();
   const tenantLocales = !isNilOrError(appConfiguration)
     ? appConfiguration.data.attributes.settings.core.locales
     : [];
   const isEmailSettingsPage = isPage('email-settings', location.pathname);
-  const isDesktopUser = isDesktop(windowSize.windowWidth);
+
+  const isSmallerThanDesktop = useBreakpoint('smallDesktop');
```

Comment:
I was wrong. I checked the implementation of `isDesktop` and we considered anything larger than tablet to be desktop. So we should rather have:
```suggestion
  const isSmallerThanTablet = useBreakpoint('tablet');
  const isDesktopUser = !isSmallerThanTablet;
```